lib/apiservers/engine/backends/container.go

```diff
@@ -1632,7 +1632,7 @@ payloadLoop:
 		c := &types.Container{
 			ID:      t.ContainerConfig.ContainerID,
 			Image:   repo,
-			Created: t.ContainerConfig.CreateTime,
+			Created: time.Unix(0, t.ContainerConfig.CreateTime).Unix(),
```

Comment:
Why do we need to limit Created to seconds?

Reply:
The (vendored) docker CLI formats the response's timestamp to a human-readable format here: https://github.com/vmware/vic/blob/c4a75ad65bc712d0181cee9570307713906d62b5/vendor/github.com/docker/docker/cli/command/formatter/container.go#L138

The `Created` field is passed as the first parameter to `time.Unix()`, which represents seconds since epoch. The nanoseconds field is a hard zero. Therefore, I placed logic to return the seconds value in `Containers()` for `docker ps` output.

Reply:
Ok, this is unfortunate that the vanilla daemon returns seconds in some cases and nanoseconds in other cases.